Gemfile

```diff
@@ -11,6 +11,9 @@ gem 'parallel_tests'
 # Debugging
 gem 'pry'                                   # Easily debug from your console with `binding.pry`
 
+# Code style
+gem 'rubocop'
```

Comment:
@deivid-rodriguez , but when I deploy to production I do not want gems like rubocop to be installed.